notebooks/byod.py

```diff
@@ -153,53 +125,76 @@ def create_cram_crai_table(table: str, listing: Iterable[str]):
 
 with callysto.Cell("markdown"):
     """
-    ## Install gsutil on your local computer using option 2 instructions in this [document](https://support.terra.bio/hc/en-us/articles/360024056512-Uploading-to-a-workspace-Google-bucket).
-    """
+    # Upload to your bucket with gsutil
 
-with callysto.Cell("markdown"):
-    """
-    ## Organize your upload to Terra
-    Using instructions from the document above, you will use the gsutil tool to upload data from your local computer to
-    your Terra workspace. To effectively use the example here, we suggest you upload the data with the suggestions
-    below.
-    """
+    ## Install gsutil as directed by this [document](https://support.terra.bio/hc/en-us/articles/360024056512-Uploading-to-a-workspace-Google-bucket#h_01EGP8GR3G10SKRXAC7H1ENXQ3).
+    Use the option "Set up gsutil on your local computer (step-by-step install)" which will allow you to upload files 
+    from your computer directly to Terra. (Those familiar with Terra are aware that files can also be uploaded by
+    simply dragging them in to the "Files" section in the Data tab, but as `gsutil` can upload more than one file at
+    a time and has additional error handling, it is recommended to use it over dragging.) But before you can upload, 
+    we will need to perform a few quick tasks.
 
```

Comment:
For the phrase "Those familiar with Terra are aware"-- I think we should avoid language assuming what a user should be familiar with as it can be invalidating. I suggest something like: "If you're uploading only a few files, you can use Terra's drag and drop feature; for many files we suggest using `gsutil` ."